src/site/markdown/server/manage-webadmin.md

```diff
@@ -1545,6 +1546,57 @@ Response codes:
  - 400: The `fromDomain` resource name is invalid
  - 400: The destination domain specified in the body is invalid
 
+## Address Mappings
+
+ - [List all address mappings](#List_all_address_mappinig)
+
+### List all address mappings
+```
```

Comment:
Skip a line above and below that code block?

Reply:
Not resolved, you didn't skip lines between the code block and you shouldn't have removed the line break above